documentation/optee_design.md

```diff
@@ -523,8 +523,104 @@ implementation, as explained in [crypto.md].
 TBD
 
 # 12. Trusted Applications
-## Format
-Trusted Applications consists of a signed ELF file.
+
+## Static and Dynamic TAs
+
+There are two ways to implement Trusted Applications, static and dynamic.
+For most cases, Dynamically-loaded TAs are preferred.
+
+### Static / Pseudo Trusted Applications
+
+These are added directly to the optee_os tree in, eg, `core/arch/arm/sta`, and
+are built along with and statically built into the OS blob.
+
+The Static / Pseudo Trusted Applications included in OP-TEE already are OP-TEE
```

Comment:
"Static / Pseudo Trusted Applications" is exactly the capitalized phrase in the section title, but okie...